practical/orbit/segmentation.groovy

```diff
@@ -34,15 +36,21 @@ println("ID:" + omeroImageId)
 // Login to create a new connection with OMERO
 ImageProviderOmero imageProvider = new ImageProviderOmero()
 imageProvider.authenticateUser(USERNAME, PASSWORD)
+Gateway gateway = imageProvider.getGatewayAndCtx().getGateway()
+SecurityContext ctx = imageProvider.getGatewayAndCtx().getCtx()
 
-// Load the model from file and classify the image
-OrbitModel model = OrbitModel.LoadFromFile(modelPath)
+List<RawAnnotation> annotations = imageProvider.LoadRawAnnotationsByType(RawAnnotation.ANNOTATION_TYPE_MODEL)
+println("Found " + annotations.size() + " files")
+
+// Load the model from first FileAnnotation and segment the image
+int fileAnnId = annotations[0].getRawAnnotationId()
+OrbitModel model = OrbitModel.LoadFromOrbit(fileAnnId)
 println("Loaded Model")
 println(model)
```

Comment:
I am afraid this prints the whole model out, but not its name. It could be the other way round.

```
selected image: com.actelion.research.orbit.imageAnalysis.components.ImageFrame[,0,0,929x758,layout=javax.swing.plaf.basic.BasicInternalFrameUI$Handler,alignmentX=0.0,alignmentY=0.0,border=javax.swing.plaf.BorderUIResource@3c9ae450,flags=33554825,maximumSize=,minimumSize=,preferredSize=java.awt.Dimension[width=0,height=0],closable=true,defaultCloseOperation=DISPOSE_ON_CLOSE,desktopIcon=javax.swing.JInternalFrame$JDesktopIcon[,0,0,140x28,invalid,layout=java.awt.BorderLayout,alignmentX=0.0,alignmentY=0.0,border=javax.swing.plaf.BorderUIResource$CompoundBorderUIResource@4338ad58,flags=16777216,maximumSize=,minimumSize=,preferredSize=],frameIcon=jar:file:/Applications/OrbitImageAnalysis-303.app/Contents/Java/orbit-image-analysis.jar!/resource/Orbital_bones32.png,iconable=true,isClosed=false,isIcon=false,isMaximum=true,isSelected=false,maximizable=true,opened=true,resizable=true,rootPane=javax.swing.JRootPane[,4,28,921x726,layout=javax.swing.JRootPane$RootLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=449,maximumSize=,minimumSize=,preferredSize=],rootPaneCheckingEnabled=true,title=77928.svs [Series 1]]
ID:14777
Found 3 files
Loaded Model
Version=12
Built with OrbitImageAnalysis version: 3.03
ClassShapes:
  Class: Background Color:(94,6,129) IncExclMode:0
  Class: Foreground Co
...
```

Reply:
yes, model.getName() would print the name.

But in general I recommend to define the model by the ID (hardcoded), not query it as the first annotation. The idea of a model is to be used by a set of files, thus let's simply define the ID.

The fileID can also change, it's a dummy file, not assigned to an image. You can store a model even without an image.

Reply:
I'm now printing the Name instead of whole model 0d2db75
We're loading the model by namespace (not linked to any particular image) simply because this is one less thing to edit when you take this code and run it in a workshop.
We can add a comment that you'd normally want to select an individual model. 
Unless there's a similarly simple way to load a model by name?

Reply:
It would be more useful to have the discernment of the models  according to ownership for our workshop (rather than by Name). Would this be possible ? Atm, the ``user-x`` is loading the first model of ``trainer-1`` instead of their own model in the script.

cc @jburel ?

Reply:
As @pwalczysko we need to distinguish the model by owner otherwise that could be very confusing